src/services/scriptingservice.cpp

```diff
@@ -288,12 +290,11 @@ void ScriptingService::initComponents() {
     clearCustomStyleSheets();
     _scriptComponents.clear();
     _settingsVariables.clear();
-    QList<Script> scripts = Script::fetchAll();
+    bool enabledOnly = true;
```

Comment:
Do we need `enabledOnly`?

Reply:
Not really, but we check for it in the loop anyways. So I thought to just fetch only the enabled scripts in the first place and remove the check inside loop

Reply:
No, I meant "do we need the variable"? 😁

Reply:
Hahah. No, but the reads like a story with it 🤣

Reply:
Clion shows me the names of method parameters. 😆 
And YOU, eager to get the last bit of a byte optimized, introduce a variable that is only used once and never changes. 😆

Reply:
It will be optimized by the compiler i am sure :wink: :laughing:

Reply:
Seeing that variable makes my head spin. Can't you just write a comment over the function call? 😁

Reply:
Okay done.